doc/user/routemap.rst

```diff
@@ -300,6 +300,11 @@ Route Map Set Command
 
    Set BGP route origin.
 
+.. index:: set table (1-4294967295)
+.. clicmd:: set tablel (1-4294967295)
+
+   Set the BGP table to a give table identfieir
```

Comment:
Let's fix the spelling and be a bit more prescriptive about what this is doing.  Namely installing the route in the given table id